django/contrib/admin/templates/admin/date_hierarchy.html

```diff
@@ -1,5 +1,9 @@
+{% load i18n %}
 {% if show %}
-<nav class="toplinks">
+<nav class="toplinks" aria-labelledby="date-hierarchy-label">
+<span id="date-hierarchy-label">
+    {% blocktranslate %}Filter by {{ field_name }}{% endblocktranslate %}
+</span> 
```

Comment:
Looked like a slight visual misalignment here, traced to a font-size delta, assuming that was not intentional, I will add some css.